### ISSUE TITLE
[Bug] Error in trained model for mobile detection

### ISSUE THREAD

**Issue by kenilaivid** [openvinotoolkit/openvino#14564]
##### System information (version)
- OpenVINO=> 2021.1.110
- Problem classification => problem in using converted models

##### Detailed description
I get this error when I run model on this **script app.py**
Link --> [https://drive.google.com/file/d/1wYY_NTUySdXdWJFFpBzwLu3z2QgpChef/view?usp=sharing](https://drive.google.com/file/d/1wYY_NTUySdXdWJFFpBzwLu3z2QgpChef/view?usp=sharing)

**Script inference.py**
Link --> [https://drive.google.com/file/d/1oeC2xO4E9ojER9seBpWVwefHjeyzv3bR/view?usp=sharing]( https://drive.google.com/file/d/1oeC2xO4E9ojER9seBpWVwefHjeyzv3bR/view?usp=sharing)
**I am able use all intel models on this script**

### Error 

![image](https://user-images.githubusercontent.com/117429458/207025135-24b87909-3294-4dd0-8fd0-1c42fa672066.png)

I am also attaching the pb file with it [https://drive.google.com/file/d/1-GHwhy8aD3x_ISKWX1hUE01NPLUrmQMu/view?usp=sharing](https://drive.google.com/file/d/1-GHwhy8aD3x_ISKWX1hUE01NPLUrmQMu/view?usp=sharing)

Converted pb file from .weights using this notebook
Weights File -->  [https://drive.google.com/file/d/1-4ODdQlDSsyPcNt-7GYljODTvhQXkN-4/view?usp=share_link](https://drive.google.com/file/d/1-4ODdQlDSsyPcNt-7GYljODTvhQXkN-4/view?usp=share_link)
Colab --> [https://drive.google.com/file/d/1sNp6u4h5EWRwHicTiwk_tTAmKZV80_q7/view?usp=sharing](https://drive.google.com/file/d/1sNp6u4h5EWRwHicTiwk_tTAmKZV80_q7/view?usp=sharing)

**Comment by Iffa-Meah** [comment]
@kenilaivid I checked your model with OpenVINO benchmark app and it seems to be supported:

Infer using image
<img width="726" alt="infer" src="https://user-images.githubusercontent.com/79643476/207254365-1beb6f28-f1f6-43f4-b1ea-d552e0c5d7b0.png">


Infer using video
<img width="733" alt="infervid" src="https://user-images.githubusercontent.com/79643476/207254416-ae62e6c4-852e-425d-88a9-32f81e6fb801.png">


Based on these findings, it seems that the issue is not coming from your model. 
You'll need to debug your custom code (app.py).

Maybe you could take one of these[ OpenVINO samples](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Samples_Overview.html) and apply accordingly.

**Comment by rkazants** [comment]
Hi @kenilaivid,

In order to unblock you with this issue now, I recommend you to try newer version of OpenVINO (for, example 2022.2) and its API 2.0. I tried your model using API 2.0: it is converted and loading into the plugin using API 2.0:

```
from openvino.runtime import Core
core = Core()
model = core.read_model("frozen_darknet_yolov4_model.xml")
compiled_model = core.compile_model(model, "CPU")
```

Please pay attention to that the input layout for API 2.0 is preserved and in NHWC layout like in TF. Here is a documentation for  transition to API 2.0: https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html

In the meantime, I tried API 1.0 and I see an issue:

```
Traceback (most recent call last):
  File "C:\Users\rkazants\Intel\github_issues\14564\run_ov_api1.py", line 4, in <module>
    network = IENetwork("frozen_darknet_yolov4_model.xml")
  File "ie_api.pyx", line 1601, in openvino.inference_engine.ie_api.IENetwork.__cinit__
ValueError: PyCapsule_GetPointer called with invalid PyCapsule object
```

@jiwaszki, @akuporos, guys don't you know what can be wrong with it?

**Comment by Iffa-Meah** [comment]
Closing issue, feel free to re-open or start a new issue if additional assistance is needed.
